crates/fluvio-types/src/lib.rs

```diff
@@ -12,7 +12,7 @@ pub use partition::PartitionError;
 //
 // Types
 //
-pub type ReplicaMap = BTreeMap<i32, Vec<i32>>;
```

Comment:
Since this change type, the crate version should be bumped to `0.4.0`.  An any other crate depend on this should be have major version bump as well

Reply:
Done